Appraisals

```diff
@@ -1,18 +1,3 @@
-# Oldest supported Rails version
-appraise "activerecord-5.2" do
-  gem "activerecord", "5.2.6"
-
-# This gemfile will be ran against ruby 2.4 and some gems might require 2.4+. The following list are the gems
-# and their respective versions that work with ruby <= 2.4
-  gem "simplecov", "0.18.5"
-  gem "docile", "1.3.5"
-end
-
-# First conflicted version
-appraise "activerecord-6.0" do
-  gem "activerecord", "6.0.0"
-end
```

Comment:
Appraisals does not support switching rubies, so the gemfile locks for those versions either work for CI rubies or for local ruby version. I think it's more important to keep running tests against those versions on CI.